### PR TITLE
#1676 - Condition to display course only with Part Time

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- added condition to display course details when offering equals to Part Time.
image

image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/programinformationrequest.json [hunk 1]
```diff
@@ -1049,12 +1049,12 @@
...
           "properties": {},
           "conditional": {
-            "show": null,
+            "show": "",
             "when": null,
             "eq": "",
             "json": ""
           },
-          "customConditional": "",
+          "customConditional": "show = data.offeringIntensitySelectedByStudent === 'Part Time'",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/programinformationrequest.json:1057]: Just a suggestion. We can have a hidden variable(isSelectedOfferingPartTime) which is calculated as `value = data.offeringIntensitySelectedByStudent === 'Part Time'` and use that.  It is just a suggestion not a blocker.

**Review by dheepak-aot — APPROVED** [review]
LGTM. 👍Add a suggestion. Not a blocker.

File: sources/packages/forms/src/form-definitions/programinformationrequest.json (hunk 1 above)
Comment (andrepestana-aot) [reply sources/packages/forms/src/form-definitions/programinformationrequest.json]: Ok. Done!

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Pull request merged**